# HG changeset patch
# User Guennadi Liakhovetski <g.liakhovetski@gmx.de>
# Date 1257793894 -3600
# Node ID d5defdb8768d9e1e951b0190b401d6b69a8b1787
# Parent  19c0469c02c3b564af14a3d2228aff51de42ccaa
v4l: add more missing linux/sched.h includes
From: Guennadi Liakhovetski <g.liakhovetski@gmx.de>

Signed-off-by: Guennadi Liakhovetski <g.liakhovetski@gmx.de>
Priority: high
---
 drivers/media/video/mx1_camera.c           |    1 +
 drivers/media/video/mx3_camera.c           |    1 +
 drivers/media/video/sh_mobile_ceu_camera.c |    1 +
 drivers/media/video/videobuf-dma-contig.c  |    1 +
 4 files changed, 4 insertions(+), 0 deletions(-)

--- a/linux/drivers/media/video/mx1_camera.c	Sat Nov 07 15:51:01 2009 -0200
+++ b/linux/drivers/media/video/mx1_camera.c	Mon Nov 09 20:11:34 2009 +0100
@@ -28,6 +28,7 @@
 #include <linux/moduleparam.h>
 #include <linux/mutex.h>
 #include <linux/platform_device.h>
+#include <linux/sched.h>
 #include <linux/time.h>
 #include <linux/version.h>
 #include <linux/videodev2.h>
--- a/linux/drivers/media/video/mx3_camera.c	Sat Nov 07 15:51:01 2009 -0200
+++ b/linux/drivers/media/video/mx3_camera.c	Mon Nov 09 20:11:34 2009 +0100
@@ -17,6 +17,7 @@
 #include <linux/clk.h>
 #include <linux/vmalloc.h>
 #include <linux/interrupt.h>
+#include <linux/sched.h>
 
 #include <media/v4l2-common.h>
 #include <media/v4l2-dev.h>
--- a/linux/drivers/media/video/sh_mobile_ceu_camera.c	Sat Nov 07 15:51:01 2009 -0200
+++ b/linux/drivers/media/video/sh_mobile_ceu_camera.c	Mon Nov 09 20:11:34 2009 +0100
@@ -31,6 +31,7 @@
 #include <linux/platform_device.h>
 #include <linux/videodev2.h>
 #include <linux/pm_runtime.h>
+#include <linux/sched.h>
 
 #include <media/v4l2-common.h>
 #include <media/v4l2-dev.h>
--- a/linux/drivers/media/video/videobuf-dma-contig.c	Sat Nov 07 15:51:01 2009 -0200
+++ b/linux/drivers/media/video/videobuf-dma-contig.c	Mon Nov 09 20:11:34 2009 +0100
@@ -19,6 +19,7 @@
 #include <linux/mm.h>
 #include <linux/pagemap.h>
 #include <linux/dma-mapping.h>
+#include <linux/sched.h>
 #include <media/videobuf-dma-contig.h>
 #include "compat.h"
 

